lib/web/mage/utils/misc.js

```diff
@@ -83,22 +83,6 @@ define([
             owner[target] = _.debounce(fn.bind(owner), limit);
         },
 
-        /**
-         * Converts mage date format to a moment.js format.
-         *
-         * @param {String} mageFormat
-         * @returns {String}
-         */
-        normalizeDate: function (mageFormat) {
```

Comment:
I think we shouldn't remove this method because someone can already use it and we will break customer implementation